xonsh/environ.py

```diff
@@ -710,7 +711,7 @@ def locate_binary(name):
 
     try:
         return next(chain.from_iterable(yield_executables(directory, name) for
-                    directory in builtins.__xonsh_env__.get('PATH') if os.path.isdir(directory)))
+                    directory in ([_get_cwd()] + builtins.__xonsh_env__.get('PATH')) if os.path.isdir(directory)))
```

Comment:
Though looking at this, the empty string case will probably fail:

```console
scopatz@localhost ~/faculty/teach uic-ds2016 $ os.path.isdir('')
False
```

This is almost certainly a bug.